toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt

```diff
@@ -120,7 +121,7 @@ public fun MapView(
     onViewpointChangedForCenterAndScale: ((Viewpoint) -> Unit)? = null,
     onViewpointChangedForBoundingGeometry: ((Viewpoint) -> Unit)? = null,
     onVisibleAreaChanged: ((Polygon) -> Unit)? = null,
-    graphicsOverlays: GraphicsOverlayCollection = rememberGraphicsOverlayCollection(),
+    graphicsOverlays: List<GraphicsOverlay> = emptyList(),
```

Comment:
I think this should be remembered so we don't create a new empty list on every recomposition.

```suggestion
    graphicsOverlays: List<GraphicsOverlay> = remember { emptyList() },
```

Reply:
@hud10837 I went back and forth about this as well and went with not remembering the emptyList() instance as in theory it would not make a difference and also that is how [google maps](https://github.com/googlemaps/android-maps-compose/blob/main/maps-compose/src/main/java/com/google/maps/android/compose/Polygon.kt#L61) had it. 
But to be safe from unintended consequences it makes sense to remember it as that would prevent us from initializing an emptyList() every time a recomposition occurs.